app/views/modals/create-secret.html

```diff
@@ -0,0 +1,16 @@
+<div class="create-secret-modal">
+  <div class="modal-header">
+    <h2>
+      Create {{type | capitalize}} Secret
+      <a ng-href="{{'creating_secrets' | helpLink}}" target="_blank"><span class="learn-more-inline">Learn more&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i></span></a>
```

Comment:
We should link to the right topic for the secret type you're creating rather than the general creating secrets topic.